pkg/option/config.go

```diff
@@ -881,6 +881,9 @@ const (
 	// XDPModeNative for loading progs with XDPModeLinkDriver
 	XDPModeNative = "native"
 
+	// XDPModeBestEffort for loading progs with XDPModeLinkDriver
+	XDPModeBestEffort = "best-effort"
```

Comment:
The additional value `best-effort` should be documented in the helm chart too. Maybe it's also worth to add the already missing value `generic`.

https://github.com/cilium/cilium/blob/main/install/kubernetes/cilium/values.yaml.tmpl#L1774-L1776

Please follow the instructions in the Cilium docs: https://docs.cilium.io/en/latest/contributing/development/dev_setup/#making-changes-to-the-helm-chart